packages/preferences-api/src/common/preference-contribution.ts

```diff
@@ -18,10 +18,21 @@ export const PreferenceSchema = Symbol("PreferenceSchema");
 export interface PreferenceSchema {
     [name: string]: Object,
     properties: {
-        [name: string]: object
+        [name: string]: PreferenceProperty
     }
 }
 
+export interface PreferenceProperty {
+    description: string;
+    type?: JsonType | JsonType[];
+    minimum?: number;
```

Comment:
Shouldn't we also have maximum as optional? I think I can see use cases where we would want a range of valid prefs.

Reply:
There are many possible properties I haven't covered here. I will add a generic signature.

Reply:
Yea I think it's fine